support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx

```diff
@@ -4,10 +4,16 @@ import { FormSection } from 'components/checkoutForm/checkoutForm';
 import DirectDebitTerms from 'components/subscriptionCheckouts/directDebit/directDebitTerms';
 import type { PaymentMethod } from 'helpers/forms/paymentMethods';
 import { DirectDebit } from 'helpers/forms/paymentMethods';
-import { privacyLink } from 'helpers/legal';
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import {
+	privacyLink,
+	subscriptionsTermsLinks,
+	supporterPlusTermsLink,
+	tier3TermsLink,
+} from 'helpers/legal';
 import { ManageMyAccountLink } from '../../pages/supporter-plus-landing/components/manageMyAccountLink';
 
-const threeTierTerms = css`
+const termsTier3 = css`
```

Comment:
```suggestion
const tierThreeTerms = css`
```

if following convention

Reply:
Agree, changed.